### PR TITLE
Misc queue from 2018 (sorry, lost sources) 

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -2383,7 +2391,9 @@ challange->challenge
 challanged->challenged
 challanges->challenges
 challege->challenge
+champain->champagne
```
Comment (arm-in) [inline codespell_lib/data/dictionary.txt]: also "champaign"

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -2317,11 +2317,14 @@ ceated->created
...
 ceating->creating
 ceation->creation
+ceding->cedeing
 cehck->check
 cehcked->checked
 cehcking->checking
 cehcks->checks
 Celcius->Celsius
+cellabration->celebration
+celler->cellar
```
Comment (arm-in) [inline codespell_lib/data/dictionary.txt]: also "seller"

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: the city?

Comment (arm-in) [reply codespell_lib/data/dictionary.txt]: You are right, there is only the city. The drink is Champagne. I mixed it up.

**Pull request closed without merging**